src/main/webapp/css/jenkins-style.css

```
@@ -22,6 +46,11 @@
 /* Font awesome icons float right                                                                                      */
 /* ------------------------------------------------------------------------------------------------------------------- */
 
+.card {
+    background-color: #fff;
```

Sorry, seems that I deleted the wrong line, shouldn't it be:
```suggestion
    background-color: var(--background);
```

BTW: How do you get rid of the IntelliJ errors for the unresolved CSS variables?

---

the proper way would be to import the css file that defines the variable but that won't work here, another option would be to pass a default through,

`var(--background, #fff)`

that should get rid of the errors but doesn't have a functional change at all